mist-client/src/main/java/edu/snu/mist/client/MISTQueryBuilder.java

```diff
@@ -60,6 +65,7 @@
   private WatermarkConfiguration getDefaultWatermarkConf() {
     return PeriodicWatermarkConfiguration.newBuilder()
         .setWatermarkPeriod(DEFAULT_WATERMARK_PERIOD)
+        .setCheckpointPeriod(DEFAULT_CHECKPOINT_PERIOD)
```

Comment:
It doesn't matter.
Actually, if it is a multiple, 2 watermarks(1 checkpoint, 1 non-checkpoint) are emitted at the same time. Do you think this can be a problem?